### ISSUE TITLE
Flathub: Add Spanish translation

### ISSUE THREAD

**Issue by manuq** [endlessm/endless-key-flatpak#99]
Metainfo is in https://github.com/endlessm/endless-key-flatpak/tree/f2eae745ca088e4b4abf88e94ae1bc3ae8fdd12d/data/metainfo

Translated screeshots here: https://www.dropbox.com/scl/fo/nxqdtyyvpkatbl0gcgqcq/h?rlkey=l524pdllrbxkkomm89qmw2ujf&dl=0

Translated copy here: https://docs.google.com/spreadsheets/d/1yVTilb6AWsyUGVncyoYSjtbOdPAEZA_BarnnTfUDxfg/edit#gid=1177927074

**Comment by dylanmccall** [comment]
Yep, translations for the metainfo file should just work, although we only _just_ updated the pot file over here: https://github.com/endlessm/endless-key-flatpak/commit/8db0bfbb470c986a063a7e5d26cc54efdf6162ec. It kind of fell by the wayside earlier. (Could do with some automation). To update pot files, you need to have a terminal open inside the build directory generated by `meson`, for example using _New Build Terminal_ in GNOME Builder, and then run this command:

```
ninja kolibri-gnome-pot
```

… and this command to merge that change with all the PO files (although that might not be necessary depending on how we talk to Transifex?):

```
ninja kolibri-gnome-update-po
```
